chat_shell/chat_shell/clients/gemini_interaction.py

```
@@ -0,0 +1,275 @@
+            GeminiInteractionError: If the API returns an error
+        """
+        url = f"{self.base_url}/v1beta/interactions/{interaction_id}"
+        params = {"stream": "true"}
+
+        logger.info(
+            "[GEMINI_CLIENT][STREAM] Request: url=%s, params=%s",
+            url,
+            params,
+        )
+
+        async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.14.14)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)